hypershift-operator/controllers/hostedcluster/internal/platform/kubevirt/kubevirt.go

```diff
@@ -0,0 +1,209 @@
+package kubevirt
+
+import (
+	"context"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	"github.com/openshift/hypershift/support/upsert"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	k8sutilspointer "k8s.io/utils/pointer"
+	capikubevirt "sigs.k8s.io/cluster-api-provider-kubevirt/api/v1beta1"
+	capiv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	hostedClusterAnnotation = "hypershift.openshift.io/cluster"
+	// TODO nargaman replace with pre-built valid image
+	imageCAPK = "registry:5000/capk:devel"
```

Comment:
:face_with_raised_eyebrow:
Please let's update this to the proper image.

Reply:
There is PR to add capk to openshift-ci, this would add the ability to generate the proper image
https://github.com/openshift/release/pull/24575

Reply:
Changed the image to one created from the fork in openshift org:
`registry.ci.openshift.org/hypershift/cluster-api-kubevirt-controller:1.0.0`